_posts/2016-12-24-crystal-0.20.3-released.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Crystal 0.20.3 released!
+summary: Crystal has a new development release.
+thumbnail: +
+author: straight-shoota
+release_notes: true
+---
+
+[Crystal 0.20.3](https://github.com/crystal-lang/crystal/releases/tag/0.20.3) has been released!
+
+As any release it includes numerous bugfixes, cool features and performance improvements - in [130 commits since 0.20.1](https://github.com/crystal-lang/crystal/compare/0.20.1...0.20.3).
+
+# Exciting Changes
+
+* `IO#gets`, `IO#each_line`, `String#lines`, `String#each_line`, etc. now chomp lines by default. You can pass `chomp: false` to prevent automatic chomping.
+* `HTTP::Handler` is no longer a class but a module.
```

Comment:
Breaking changes should be noted as breaking.

Reply:
There are a lot of them... I don't think we need that here.  I didn't even include all breaking changes in the blog posts, so what's the point in highlighting those that are mentioned?

Reply:
I think breaking changes should always be mentioned everywhere we published release notes. Other than that the PR is great!